cli/azd/pkg/infra/provisioning/provisioning_progress_display.go

```diff
@@ -142,12 +143,22 @@ func (display *ProvisioningProgressDisplay) logNewlyCreatedResources(
 		// Don't log resource types for Azure resources that we do not have a translation of the resource type for.
 		// This will be improved on in a future iteration.
 		if resourceTypeDisplayName != "" {
+			var duration time.Duration
+			if resource.Properties.Duration != nil {
+				if p, err := period.Parse(*resource.Properties.Duration); err == nil {
```

Comment:
I really love the direction here. Timings was something we were thinking about a lot when we initially implemented the display. 

From the screenshot you provided, I wonder if users will ask "where did the time go?" Since it's not clear that the time was spent on sub-resources that we don't currently capture in the output.

So I wonder if we should we be rolling sub-resources into the "resource provisioning" operations we display? That way we can attribute all-up inclusive time spent on each resource in the display. 

EDIT: I'm not sure if this is possible if we're simply discovering deployment objects. Do we know the complete set  beforehand? If not, it's possible that deployments are waiting to be spawned from other dependent deployments. So querying this may not be feasible.

Reply:
I do understand now with how you're selectively displaying the timing output helps highlight "long running" operations. If so, the distribution of time is less important here.

Reply:
> EDIT: I'm not sure if this is possible if we're simply discovering deployment objects. Do we know the complete set beforehand? If not, it's possible that deployments are waiting to be spawned from other dependent deployments. So querying this may not be feasible.

Yes - this is common.  For example, if you look at the root level subscription deployment for say the ToDo application, you'll see a bunch of nested deployments roll in as the overall deployment continues (for example, the nested deployment that will make the API service will not show up until after the nested deployment that provisions the KeyVault completes, since it needs outputs of that deployment for that operation to be scheduled).

Reply:
Rolling back this part of the diff for now since we don't want to show per resource timing information at this level.